tests/scripts/get_test_files.py

```diff
@@ -0,0 +1,60 @@
+"""
```

Comment:
why do you need it in a separate file, seems like it could live in the nox file, it is only used in one place

Reply:
This needs to be in separate file because nox executes the test gathering in a separate virtual environment. So you need to call the nox `session.run` which only runs commands as if it was on the cli.

Reply:
in that case should we move it to the tools folder?

Reply:
It could be in the tools folder. I felt like keeping it closer to the these area made more sense to me since the only real purpose is for CircleCI.

But I don't have a strong preference.